### ISSUE TITLE
warn on closing terminal request if multiple processes running

### ISSUE THREAD

**Issue by christianparpart** [christianparpart/contour#208]
Some terminals warn off a close request recent happened but there's are still more processes running than the initially slasher shell.

See if we can use `tcgetpgrp()`

**Comment by Yaraslaut** [comment]
Makes sense especially now when multiple tabs can be opened, but we also need to make it configurable, and turned on by default
